### DISCUSSION TITLE
Rotation Order for Maya Reference Nodes

### DISCUSSION THREAD

**Discussion by mwalk176** [Autodesk/maya-usd#3376] [Q&A]
Hi all, running into an issue that maybe someone on here could answer, but is it possible to change the rotation order on a maya reference node that is a child of a USD Prim? It's currently locked and reports this error: 

`# Error: RuntimeError: file <maya console> line 13: Node is locked to maintain its USD hierarchy. Attribute 'testPrim_MayaReference.rotateOrder' is from a locked node, so it cannot be unlocked.`

What do I need to do to be able to change the rotation order from xyz to yzx? Is there something in USD that MayaUsd is looking at that I can change so the rotation order will update?

**Comment by mwalk176** [comment]
Has anyone had any experience with this? It's still a blocker for us at our studio

**Reply by womanyoyoyo-adsk** [reply]
Hi @mwalk176,

I know that error but it shouldn't be popping up for rotation order. Do you mind attaching repro steps with the Maya Reference being just a simple mesh?

Thanks,
Natalia
